### ISSUE TITLE
Feature Request - special message dispay

### ISSUE THREAD

**Issue by N6RFM** [f4gkr/PicTalk#19]
Hi Sylvain,

From time to time, PicSat is sending "special hidden messages".  Perhaps in future, there will be a feature from within PicSat to display these directly.  This may already be taken care of when a method to export KSS data is included using third party software such as DK3WN.

73,

Bob

**Comment by f4gkr** [comment]
Yep ;-) Yesterday during my demo in the Electrolab hackerspace we had fun
with a special message and had to log in the PicSat server to see them ....
Good suggestion !

2018-02-25 17:04 GMT+01:00 N6RFM <notifications@github.com>:

**Comment by N6RFM** [comment]
Yes, watched with interest the lectures from both you and Mathias!  Je parle seulement un très petit français, donc c'était difficile de suivre, sauf pour une idée générale de ce qui se disait.  :-)

**Comment by f4gkr** [comment]
:-) parfait

2018-02-25 17:19 GMT+01:00 N6RFM <notifications@github.com>:
